### PR TITLE
fix: Switched from displaying our guid to users external IDs

### LINKED ISSUE #47: chore(MapLocation Table): update to include externalId and omit guid
## Summary

We have tables in our UI in ~two~ three places:

1. Defining locations for a map
2. After uploading a file
3. When viewing a map

## Desired Behaviour

The most valuable thing to a user is likely the `externalId` field. Currently we're only exposing that when viewing the map, and in other places we are showing the "behind the scenes" guid we create for each MapLocation.

We should omit `guid` and show users the `externalId` and `title`'s of MapLocations.

### PR DESCRIPTION
**Pull request by madecki**
For locations tables.

Closes #47

### CODE REVIEW

File: nerdlets/geo-ops-nerdlet/MapLocationFilesUpload.js [hunk 1]
```diff
@@ -258,8 +253,8 @@ export default class MapLocationFilesUpload extends React.Component {
   getColumns() {
     const columns = [
       {
-        dataField: 'guid',
-        text: 'GUID',
+        dataField: 'externalId',
+        text: 'ExID',
```
Comment (norbertsuski) [inline nerdlets/geo-ops-nerdlet/MapLocationFilesUpload.js:257]: Maybe you can put here 'EXID' just like on a map view? Also all other headers are uppercased

File: nerdlets/shared/components/LocationTable.js [hunk 2]
```diff
@@ -32,8 +32,8 @@ export default class MapLocationTable extends Component {
   getColumns() {
     return [
       {
-        dataField: 'document.guid',
-        text: 'GUID',
+        dataField: 'document.externalId',
+        text: 'ExID',
```
Comment (norbertsuski) [inline nerdlets/shared/components/LocationTable.js:36]: Maybe you can put here 'EXID' just like on a map view? Also all other headers are uppercased

Comment (madecki) [reply nerdlets/shared/components/LocationTable.js:36]: I used existing convention from other parts of the system but don't worry - it's uppercased then rendered :)

Comment (norbertsuski) [reply nerdlets/shared/components/LocationTable.js:36]: Ok :)

**Comment by tangollama** [comment]
Great. I've asked @devfreddy to give it a quick look and then we'll merge.

**Comment by devfreddy** [comment]
@tangollama LGTM 👍

**Pull request merged**